src/main/resources/com/google/api/codegen/java/http_unit_test.snip

```diff
@@ -0,0 +1,132 @@
+@extends "java/common.snip"
+@extends "java/initcode.snip"
+
+@snippet generate(xapiTest)
```

Comment:
This file was committed already as http_test.snip, right?

Reply:
whoops this snuck in. deleted now.